.circleci/config.yml

```diff
@@ -21,6 +21,13 @@ jobs:
 
       - protobuf/install
 
+      - run:
+          name: "Install protoc plugin"
```

Comment:
This is the download for `grpc-web` protobuf generator. This was not needed with grpc-node as the default protoc generator is installed via npm in the grpc-tools library. `node_modules/grpc-tools/bin/protoc`

Reply:
It would make or build process easier but I think sticking to the official implementation for this PR just to get it out there would make sense. We can revisit this later if it is impacting our Developer experience.